### PR TITLE
Add new file: take-more-nap.md

### PR DESCRIPTION
**Pull request by silvercanopus**

### CODE REVIEW

**Comment by silvercanopus** [comment]
Please review this, @casey-collab

File: english/take-more-nap/take-more-nap.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You woke up a couple of hours later, feeling refreshed. The two malt balls are still there.
```
Comment (casey-collab) [inline english/take-more-nap/take-more-nap.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by silvercanopus** [comment]
How about now, @casey-collab ?

**Comment by casey-collab** [comment]
Looks great! :wind_chime:

**Pull request merged**